nvflare/fuel/hci/client/overseer_service_finder.py

```diff
@@ -0,0 +1,48 @@
+from nvflare.apis.overseer_spec import OverseerAgent
+from nvflare.fuel.hci.reg import CommandModule
+from nvflare.ha.ha_admin_cmds import HACommandModule
+
+from .api_spec import ServiceFinder
+
+
+class ServiceFinderByOverseer(ServiceFinder):
```

Comment:
We should be able to just use `ServiceFinderByOverseer` under all circumstances since we already have `DummyOverseerAgent`?

That means we don't need rr_service_finder and static_service_finder.

Since we already have OverseerAgent general concept that is for discovering the service, maybe we don't need to introduce ServiceFinder concept?

Reply:
Leave this for future discussion.